src/ProductTour/Checkpoint.scss

```
@@ -86,13 +53,18 @@
   .pgn__checkpoint-header {
     display: flex;
     justify-content: space-between;
-    margin-bottom: map.get($spacers, 2\.5);
+    margin-bottom: map.get($spacers, 2);
```

non-blocking since this was using `map.get` before this change, but it could be nice to change this to

```suggestion
    margin-bottom: var(--pgn-spacing-spacer-2);
```

that being said, since there seem to be quite a few of these still hanging around, it probably makes more sense to just do a sweep separate from this sync.

---

Yeah, I had a similar thought and deferred on that additional refactoring so we tackle it separately in a standalone contribution.